source/Examples/SmartCache/stdafx.h

```diff
@@ -6,15 +6,19 @@
 // @File: stdafx.h
 //
 // Purpose:
-//      <description>
+//  This is the standard include header, expected to be in every compilation unit
+//  in this project.
```

Comment:
This is not as common for non-Windows devs, so it seemed usful to explain a tad.